src/components/new-print-page/printContext.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useReducer, useEffect } from "react";
+let currentDate = new Date();
+const electrodeBatchDate= ""+currentDate.getMonth()+((currentDate.getFullYear()).toString().slice(-2));
```

Comment:
Couple of things here:

- You could assign `((currentDate.getFullYear()).toString().slice(-2));` to a variable to describe what it is.  
- You could use a template string to combine the variables together

Reply:
i just used the package "moment.js"